pgdrive/pg_config/parameter_space.py

```diff
@@ -105,3 +105,7 @@ class BlockParameterSpace:
         Parameter.length: PGBoxSpace(min=20, max=40),  # accelerate/decelerate part length
         Parameter.lane_num: PGDiscreteSpace(2)
     }
+    BOTTLENECK_PARAMETER = {
+        Parameter.length: PGBoxSpace(min=20, max=20),  # the length of straigh part
+        Parameter.lane_num: PGDiscreteSpace(2),  # the lane num increased or descreased now fix to 1
```

Comment:
Should we allow it to be 1?

Reply:
It is hard to decide how much lane num to change, so now I only allow to increase or decrease 1-2 lanes